maker/src/cfd.rs

```diff
@@ -239,6 +239,7 @@ impl<O, T, W> Actor<O, T, W> {
         }
     }
 
+    #[instrument]
```

Comment:
does this compile? I tried to compile from scratch and it seems like it needs `tracing::instrument`. we might want to also `skip(self)` here as otherwise we'll get a lot of info that's pretty useless for us (maker actor does not change that much)

Reply:
Ah yes, my mistake. I'll add it now.

We can add skip, yes, but if it isn't really worth debug printing, maybe we could also skim down the debug implementation itself so that we don't need to remember to skip it all the time? This might be detrimental if we do actually want to print it. The other option is to remove the debug derive if it's unused except here.

Reply:
I don't think we have `Debug` implemented for this actor at all? I don't mind really basic debug impl, but in this case, I don't see any point in printing this info. we can add it later on.

Reply:
Addressed now